### PR TITLE
ack docs + param bugfix in ack lua module

### PR DESCRIPTION
**Pull request by antonhornquist**

### CODE REVIEW

File: docs/jah/ack/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+- 8 voice sample playback of mono or stereo samples
+- Sample start / end and loop positions
+- Resonant multimode filter
+- Envelopes for volume and filter cutoff
+- Master delay and reverb
+
+## Commands
+
+_Note: channels are indexed 0-7 in commands._
```
Comment (pq) [inline docs/jah/ack/index.md]: aside: curious why not 1-8?

File: docs/jah/ack/index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+### Triggering
+
+- trig i: triggers single channel given a channelnum.
+- multiTrig iiiiiiii: triggers multiple channels. one 1/0 argument for each channel. 1 means trigger.
+
+### Channel Settings
+
+First argument is always channelnum.
```
Comment (pq) [inline docs/jah/ack/index.md]: nit (feel free to ignore!):

some settings descriptions end with a `.` and some don't.  if you're finicky, maybe make them consistent?

File: docs/jah/ack/index.md (hunk 1 above)
Comment (antonhornquist) [reply docs/jah/ack/index.md]: good question. i dunno :) other than the fact that sc typically start indices with 0 and lua with 1.

File: docs/jah/ack/index.md (hunk 2 above)
Comment (antonhornquist) [reply docs/jah/ack/index.md]: thanks, hope to revise these a bit later today

File: docs/jah/ack/index.md (hunk 1 above)
Comment (antonhornquist) [reply docs/jah/ack/index.md]: also, looking at other monome osc protocols (serialosc) indices start with 0.

Comment (pq) [reply docs/jah/ack/index.md]: 👍 

lua's outsider status strikes again!  (where's the `:punk:` emoji?)

anyway, something to consider making crystal clear in the general docs (fyi @tehn)

Comment (catfact) [reply docs/jah/ack/index.md]: fwiw, in other engines that take index arguments, i've been keeping them 1-based and shifting them at the receiving end (in sclang)

Comment (pq) [reply docs/jah/ack/index.md]: oh. cool!  i wonder if we shouldn't just adopt that as a convention?

Comment (antonhornquist) [reply docs/jah/ack/index.md]: yeah if we establish best practices / naming standards on engines i'll change to that. right now i do the shift in the generic `dust/lib/jah/ack.lua` module.

**Pull request merged**